doc/ImplementingApplications/ConceptOfApplicationPatternPackage/ConceptOfApplicationPatternPackage.md

```diff
@@ -14,44 +14,42 @@ This library provides APIs,
 
 ## Application Pattern Modules
 
-ApplicationPattern modules consists of the following sub-modules
```

Comment:
Lines 1 to 14 are overlapping with content from ApplicationPatternDcoumentation . We can modify the documentation as below : 

1) remove the lines from 1 to 14 in this document.
2) rename the document to ApplictionPatternModulesOverview

Reply:
Moved the lines from this file to ApplicationPatternDocumentation (the ones in that document were outdated).